HousingSearchApi.Tests/V1/E2ETests/Fixtures/ProcessFixture.cs

```diff
@@ -0,0 +1,96 @@
+    {
+        public const string INDEX = "processes";
+        private static readonly Fixture _fixture = new Fixture();
+
+        public static ProcessStub[] Processes =
+        {
+            new ProcessStub{ Id = Guid.NewGuid(), TargetId = Guid.NewGuid(), TargetType = TargetType.tenure, ProcessName = ProcessName.soletojoint, State = "ProcessStarted", PatchAssignment = _fixture.Create<PatchAssignment>(), RelatedEntities = _fixture.CreateMany<RelatedEntity>().ToList()},
+            new ProcessStub{ Id = Guid.NewGuid(), TargetId = Guid.NewGuid(), TargetType = TargetType.person, ProcessName = ProcessName.changeofname, State = "ProcessUpdated", PatchAssignment = _fixture.Create<PatchAssignment>(), RelatedEntities = _fixture.CreateMany<RelatedEntity>().ToList()},
+            new ProcessStub{ Id = Guid.NewGuid(), TargetId = Guid.NewGuid(), TargetType = TargetType.asset, ProcessName = ProcessName.soletojoint, State = "ProcessClosed", PatchAssignment = _fixture.Create<PatchAssignment>(), RelatedEntities = _fixture.CreateMany<RelatedEntity>().ToList()},
+            new ProcessStub{ Id = Guid.NewGuid(), TargetId = Guid.NewGuid(), TargetType = TargetType.person, ProcessName = ProcessName.changeofname, State = "ProcessCancelled", PatchAssignment = _fixture.Create<PatchAssignment>(), RelatedEntities = _fixture.CreateMany<RelatedEntity>().ToList()},
+            new ProcessStub{ Id = Guid.NewGuid(), TargetId = Guid.NewGuid(), TargetType = TargetType.tenure, ProcessName = ProcessName.soletojoint, State = "ProcessCompleted", PatchAssignment = _fixture.Create<PatchAssignment>(), RelatedEntities = _fixture.CreateMany<RelatedEntity>().ToList()},
+        };
+
+
+        public ProcessFixture(IElasticClient elasticClient, HttpClient httpClient) : base(elasticClient, httpClient)
+        {
+
+            WaitForESInstance();
+        }
+
+        public void GivenAnProcessIndexExists()
+        {
+            ElasticSearchClient.Indices.Delete(INDEX);
+
+            if (!ElasticSearchClient.Indices.Exists(Indices.Index(INDEX)).Exists)
+            {
+                var processSettingsDoc = File.ReadAllTextAsync("./data/elasticsearch/processesIndex.json").Result;
+                ElasticSearchClient.LowLevel.Indices.CreateAsync<BytesResponse>(INDEX, processSettingsDoc)
+                    .ConfigureAwait(true);
+
+                var processes = CreateProcessData();
+                var awaitable = ElasticSearchClient.IndexManyAsync(processes, INDEX).ConfigureAwait(true);
+
+                while (!awaitable.GetAwaiter().IsCompleted)
+                {
+
+                }
+
+                Thread.Sleep(5000);
+            }
+        }
+
+        private List<QueryableProcess> CreateProcessData()
+        {
+            var listOfProcesses = new List<QueryableProcess>();
+
+            foreach (var value in Processes)
+            {
+                var process = _fixture.Create<QueryableProcess>();
+                process.Id = value.Id;
```

Comment:
Is there a factory method you could use to convert from QueryableProcess to Process instead?

Reply:
There isn't any and they how the other tests are completed. Maybe a future refactor that we can do